dotCMS/src/main/java/com/dotcms/telemetry/collectors/api/ApiMetricWebInterceptor.java

```diff
@@ -32,128 +44,200 @@ public String[] getFilters() {
 
     @Override
     public Result intercept(HttpServletRequest req, HttpServletResponse res) throws IOException {
-        return new Result.Builder().next()
-                .wrap(new RereadInputStreamRequest(req))
-                .build();
+        final String requestURI = req.getRequestURI();
+        
+        // Only process API requests
+        if (!requestURI.contains("/api/")) {
```

Comment:
the interceptors already does that on the getFilters, you can limited the request scope right there